crates/astria-core/src/bridge.rs

```diff
@@ -12,3 +14,18 @@ pub struct Ics20WithdrawalFromRollupMemo {
     pub block_number: u64,
     pub transaction_hash: [u8; 32],
 }
+
+/// Memo format for a ICS20 transfer to Astria which is sent to a
+/// bridge account, which will then be deposited into the rollup.
+#[derive(Debug, Clone)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize),
+    derive(serde::Deserialize)
+)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct Ics20TransferDepositMemo {
+    /// the destination address for the deposit on the rollup
+    #[serde(rename = "rollupAddress")]
```

Comment:
The rename here is unnecessary becase `rename_all = "camelCase"` should cover this already.